examples/usage.sh

```diff
@@ -22,3 +22,16 @@ head -n 20 output.xyz
 # All command line flags of the eval sub-command can be listed via
 
 metatensor-models eval --help
+
+# However, before we export the model, we need to run the following command to
+# set up the environment for exporting
+
+python ../scripts/setup.py
+
+# The above script can be found in the `scripts` folder of the repository.
+
+# Finally, the `metatestor-models export`, i.e.,
```

Comment:
Should we maybe add a comment that even though the file ending of `model.pt` and `exported-model.pt` is the same, the organization of the content is different. The first one is the internal format which allows for retraining while the latter is  a model in evaluation mode, compiled (?) functions which can only be used for running md.

Question though. Can one use the one also for the eval script? If no we should add a check and an error message in the eval script.

Reply:
Both should be usable for eval

Reply:
Okay makes sens to add this maybe here.